server/src/main/java/org/elasticsearch/index/mapper/ParsedDocument.java

```diff
@@ -67,17 +101,6 @@ public void updateSeqID(long sequenceNumber, long primaryTerm) {
         this.seqID.primaryTerm.setLongValue(primaryTerm);
     }
 
-    /**
-     * Makes the processing document as a tombstone document rather than a regular document.
-     * Tombstone documents are stored in Lucene index to represent delete operations or Noops.
-     */
-    ParsedDocument toTombstone() {
-        assert docs().size() == 1 : "Tombstone should have a single doc [" + docs() + "]";
```

Comment:
do we still have this assertion somewhere?

Reply:
I don't really think it's necessary because we're completely controlling the creation of these docs, but I've added it back to the `tombstone` method.